components/query_handler/queryFunctions.py

```
@@ -0,0 +1,134 @@
+import json
```

Ruff (0.8.2)

`json` imported but unused

Remove unused import: `json`

(F401)